data/root/config

```diff
@@ -1,4 +1,4 @@
-root:			common root cracklib-dict-full.rpm bind ?yast2-trans-<lang>.rpm
+root:			common root libstoragemgmt cracklib-dict-full.rpm bind ?yast2-trans-<lang>.rpm
```

Comment:
This means libstoragemgmt will always be loaded, which is not your intention, iirc. Just ,leave the line unchanged.